### ISSUE TITLE
[Panel] Deleting a page: Redirect to parent page does not work for page depth > 2

### ISSUE THREAD

**Issue by SebastianEberlein** [getkirby/kirby#861]
I used the Beta 4 Plainkit and the latest nightly build:

After I delete a page that is sub-subpage, I am not redirected to the parent page. 

The redirect works for a page like `/portfolio/projects`, but not for `/portfolio/projects/project-a`.

Here’s a gif where I add and delete a sub-subpage. At the end I reload the page.

![delete](https://user-images.githubusercontent.com/15672385/43887956-91ca9766-9bc0-11e8-9642-03e5a1ab84e5.gif)

**Comment by bastianallgeier** [comment]
✅

**Comment by johannahoerrmann** [comment]
I just experienced something similar to this (without an error though) that threw me off: After deleting a subpage (level 3), I am directed to Site. I would expect to be directed to the parent page (level 2).

Kirby version:
3.0.0-RC-4.1

**Comment by johannahoerrmann** [comment]
Oh, and when I use my browser back button then I land on the non-existing page I just deleted (expected), and using the back button once more, I land on the level 2 page which shows the same error:

<img width="1288" alt="grafik" src="https://user-images.githubusercontent.com/149678/50966343-f0f94100-14d4-11e9-9896-9460e4db5f66.png">

Reloading solves this issue.

**Comment by SebastianEberlein** [comment]
I can reproduce Johannas issue.

Kirby version:
3.0.0-RC-4.0

**Comment by afbora** [comment]
I have same issue on Kirby 3.2.3. When i deleted the level 3 page, redirecting to panel dashboard instead of parent (Level 2) with no error in console.

**Comment by afbora** [comment]
@distantnative @lukasbestle After confirming the issue, can we reopen the issue or move it to the new one please?

**Comment by texnixe** [comment]
Yes, I can confirm this. Let's create a new issue.
